components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/internal/http/api/Constants.java

```diff
@@ -30,4 +30,6 @@
     public static final String INTERNAL_APIS_FILE = "internal-apis.xml";
     public static final String PREFIX_TO_ENABLE_INTERNAL_APIS = "enable";
 
+    public static final String TAG_HOT_DEPLOYMENT = "hotdeployment";
```

Comment:
Do we use this constant? If not shall we remove this?

Reply:
Done.